src/engine/bank01.asm

```diff
@@ -8142,12 +8142,154 @@ _TossCoin: ; 71ad (1:71ad)
 	ret
 ; 0x72ff
 
-	INCROM $72ff, $7354
+Func_72ff: ; 72ff (1:72ff)
+	ldh [hff96], a
+	ld a, [wDuelType]
+	cp DUELTYPE_LINK
+	ret nz
+	ldh a, [hff96]
+	call SerialSendByte
+	call Func_7344
+	ret
+; 0x7310
+
+Func_7310: ; 7310 (1:7310)
+	ldh [hff96], a
+	ld a, [wDuelType]
+	cp $01
+	jr z, .asm_7338
+.asm_7319
+	call DoFrame
+	call CheckAnyAnimationPlaying
+	jr c, .asm_7319
+	ldh a, [hff96]
+	ret
+	ldh [hff96], a
+	ld a, [wDuelType]
+	cp DUELTYPE_LINK
+	jr z, .asm_7338
+	ld a, $1e
+.asm_732f
+	call DoFrame
+	dec a
+	jr nz, .asm_732f
+	ldh a, [hff96]
+	ret
+.asm_7338
+	call DoFrame
+	call SerialRecvByte
+	jr c, .asm_7338
+	call Func_7344
+	ret
+; 0x7344
+
+Func_7344: ; 7344 (1:7344)
+    push af
+    ld a, [wSerialFlags]
+    or a
+    jr nz, .asm_734d
+    pop af
+    ret
+.asm_734d
+    call Func_3b31
+    call DuelTransmissionError
+    ret
+; 0x7354
 
 BuildVersion: ; 7354 (1:7354)
 	db "VER 12/20 09:36", TX_END
 
-	INCROM $7364, $7415
+Func_7364: ; 7364 (1:7364)
+	xor a
+	ld [wTileMapFill], a
+	call ZeroObjectPositionsAndToggleOAMCopy
+	call EmptyScreen
+	call LoadSymbolsFont
+	ld de, $389f
+	call SetupText
+	call DrawWideTextBox
+	call EnableLCD
+	xor a
+	ld [wOpponentDeckID], a
+	call Func_73d8
+.asm_7384
+	call DoFrame
+	ldh a, [hDPadHeld]
+	or a
+	jr z, .asm_7384
+	ld b, a
+	and $09
+	jr nz, .asm_73cd
+	bit 1, b
+	jr nz, .asm_73cb
+	ld a, [wOpponentDeckID]
+	bit 4, b
+	jr z, .asm_73a2
+	inc a
+	cp $35
+	jr c, .asm_73a2
+	xor a
+.asm_73a2
+	bit 5, b
+	jr z, .asm_73ae
+	or a
+	jr nz, .asm_73ad
+	ld a, $34
+	jr .asm_73ae
+.asm_73ad
+	dec a
+.asm_73ae
+	bit 6, b
+	jr z, .asm_73b9
+	add $0a
+	cp $35
+	jr c, .asm_73b9
+	xor a
+.asm_73b9
+	bit 7, b
+	jr z, .asm_73c3
+	sub $0a
+	jr nc, .asm_73c3
+	ld a, $34
```

Comment:
Since you can see this value is written to `wOpponentDeckID`, you should look for what constant can be used here instead (from `src/constants/deck_constants.asm. Be careful to read the comments from there because deck constants are a little confusing.)

Reply:
Please read the comments in `src/constants/deck_constants.asm` carefully. The value being written to `wOpponentDeckID` should use the `_ID` suffix. Look for other places in the code where `wOpponentDeckID` is being written to to see what I mean.